src/lib/theme/default.ts

```diff
@@ -491,12 +491,12 @@ const theme: FlowbiteTheme = {
   },
   listGroup: {
     root: {
-      base: 'list-none rounded-lg border border-gray-200 bg-white text-sm font-medium text-gray-900 dark:border-gray-600 dark:bg-gray-700 dark:text-white text-left',
+      base: 'list-none rounded-lg border border-gray-200 bg-white text-sm font-medium text-gray-900 dark:border-gray-600 dark:bg-gray-700 dark:text-white',
     },
     item: {
-      base: '[&>*]:first:rounded-t-lg [&>*]:last:rounded-b-lg [&>*]:last:border-b-0',
+      base: 'flex w-full cursor-pointer border-b border-gray-200 py-2 px-4 first:rounded-t-lg last:rounded-b-lg last:border-b-0 dark:border-gray-600',
       link: {
-        base: 'flex w-full border-b border-gray-200 py-2 px-4 dark:border-gray-600',
+        base: '',
```

Comment:
This one seems to be unrelated, and it is reverting https://github.com/themesberg/flowbite-react/pull/668. Is it intentional?